migrating_from_ocp_3_to_4/index.adoc

```diff
@@ -0,0 +1,86 @@
+. xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#configuring-replication-repository_installing-3-4[Configure object storage to use as a replication repository].
+
+[id="mtc-overview-upgrade-mtc"]
+== Upgrading MTC
+You can xref:../migrating_from_ocp_3_to_4/upgrading-3-4.adoc#upgrading-3-4[upgrade the {mtc-full} ({mtc-short})] on {product-title} {product-version} by using OLM. You can upgrade {mtc-short} on {product-title} 3 by reinstalling the legacy {mtc-full} Operator.
+
+[id="mtc-overview-mtc-checklists"]
+== Reviewing MTC checklists
+Before you migrate your application workloads with the Migration Toolkit for Containers (MTC), review the xref:../migrating_from_ocp_3_to_4/premigration-checklists-3-4.adoc#premigration-checklists-3-4[premigration checklists].
+
+[id="mtc-overview-migrate-mtc-applications"]
+== Migrating applications
+You can migrate your applications by using the MTC xref:../migrating_from_ocp_3_to_4/migrating-applications-3-4.adoc#migrating-applications-mtc-web-console_migrating-applications-3-4[web console] or xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migrating-applications-cli_advanced-migration-options-3-4[the command line].
+
+[id="mtc-overview-advanced-migration-options"]
+== Advanced migration options
+You can automate your migrations and modify the `MigPlan` and `MigrationController` custom resources in order to perform large-scale migrations and to improve performance. You can check the following items:
+
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-creating-registry-route-for-dim_advanced-migration-options-3-4[Create a registry route for direct image migration]
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-configuring-proxies_advanced-migration-options-3-4[Configuring proxies]
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-migrating-applications-api_advanced-migration-options-3-4[Migrating an application by using the MTC API]
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-state-migration-cli_advanced-migration-options-3-4[Running a state migration]
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-hooks_advanced-migration-options-3-4[Creating migration hooks]
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-plan-options_advanced-migration-options-3-4[Editing, excluding, and mapping migrated resources]
+* xref:../migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc#migration-controller-options_advanced-migration-options-3-4[Configuring the migration controller for large migrations]
+
+[id="mtc-overview-troubleshooting-mtc"]
+== Troubleshooting migrations
+You can perform the following troubleshooting tasks:
+
+* xref:../migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc#migration-viewing-migration-plan-resources_troubleshooting-3-4[Viewing plan resources]
```

Comment:
"Viewing plan resources" > "Viewing migration plan resources by using the MTC web console"

Reply:
acked